dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops

```diff
@@ -1633,3 +1633,11 @@ redirect.location.cannot.be.null.useraction=Update the redirect location by spec
 sendRedirect.hyper.text.url=<html><body><p>Redirecting to <a href="{0}"></a></p></body></html>
 
 query.or.post.is.null=Request query string or post data is null.
+
+read.write.bytebuffer.null=CWWWC0009E: An attempt to read from an InputStream or write to an OutputStream failed because a variable argument ByteBuffer was null.
+read.write.bytebuffer.null.explanation=The variable argument ByteBuffer passed to the read or write API must not be null.
+read.write.bytebuffer.null.useraction=Modify the application code according to the error explanation.
+
+read.write.failed.isReady.false=CWWWC0010E: An attempt to read from an InputStream or write to an OutputStream failed because isReady API returns false.
```

Comment:
```suggestion
read.write.failed.isReady.false=CWWWC0010E: An attempt to read from an InputStream class or write to an OutputStream class failed because the isReady API returned false.
```